src/main/java/seedu/attendance/AttendanceList.java

```diff
@@ -16,33 +16,67 @@
 public class AttendanceList {
 
     protected ArrayList<Attendance> attendanceList;
```

Comment:
This attribute doesn't seem to be used at all... Perhaps can consider removing it?

Reply:
it is used, in fetching the attendance list.